rstudio/rstudio
# added global option to disable clickable hyperlinks (fixes #6689)

### Intent

This fixes issue #6689 that I raised back in April, adding an option to disable comments from automatically creating hyperlinks when URLs are added. See new option circled:
![Screen Shot 2020-10-05 at 14 33 30](https://user-images.githubusercontent.com/2408155/95123666-e99db080-0717-11eb-826b-6bce9191095a.png)

### Approach

I had a lot of guidance for this feature; see comments on #6689.

### QA Notes

Should be able to add a comment with any URL, and toggle the option to test. Also close and re-open RStudio & the setting should preserve its value.
